static/js/components/GameRoom.js

```diff
@@ -51,33 +51,33 @@ export class GameRoom {
         };
 
         this.gameSocket.onmessage = e => {
-            let data = JSON.parse(e.data)["payload"];
-            if (data["move_type"] == "NEXT_ROOM") {
+            let message = JSON.parse(e.data)["payload"];
+            if (message["move_type"] == "NEXT_ROOM") {
```

Comment:
prefer === in JS. Comment applies throughout.

Reply:
updated all instances in GameRoom.js